windows/test-app.js

```diff
@@ -207,6 +230,31 @@ function copyProjectTemplateAndReplace(
   }
 
   const solutionFileDestPath = path.join(destPath, windowsDir);
+
+  const testProjectFilePath = path.join(
+    solutionFileDestPath,
+    "ReactTestAppTests"
+  );
+  const testProjectTemplateFile = "ReactTestAppTestsTemplate.vcxproj";
+  let testProjectEntry = "";
+
+  if (fs.existsSync(path.join(testProjectFilePath, testProjectTemplateFile))) {
```

Comment:
This is a bit weird check. Should we be checking whether `ReactTestAppTests\ReactTestAppTests.vcxproj` exists rather than `ReactTestAppTestsTemplate.vcxproj`?

Reply:
Yes, I agree that it's weird. The reason we do it is that there are a couple of paths that need to be dynamically resolved in `ReactTestAppTests.vcxproj`, so what happens here is that it gets created from the template. If you have any suggestions on how to improve it, I would appreciate it.

Reply:
Let's leave this as is for now. I want to improve the overall story for adding test projects in #204.